docs/source/index.rst

```diff
@@ -70,24 +70,28 @@ Details
 Change Log
 ==========
 
-Version 0.1.5
------------
+Version 0.4.0
+-------------
+- Ignore DICOMDIR files
```

Comment:
```suggestion
- Ignore DICOMDIR files
- Allow HighBit and BitsStored attributes to be non-uniform when combining slices
- Support combining channeled slices
- Drop support for Python 3.4
```

Reply:
I added a couple of other items as well in a direct commit.